src/main/java/com/epam/reportportal/rules/exception/ErrorType.java

```diff
@@ -0,0 +1,526 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.reportportal.rules.exception;
+
+import lombok.Getter;
+
+/**
+ * Report Portal's exception list
+ *
+ * @author Andrei Varabyeu
+ */
+@Getter
+public enum ErrorType {
+
+  /**
+   * Incorrect Report Portal WS Request
+   */
+  INCORRECT_REQUEST(4001, "Incorrect Request. {}"),
```

Comment:
Let's try to leave error types used only in this repo

Reply:
done